[Billing] Document how the default plan works (#2981)
clerkinc/clerk-docs: docs/guides/billing/default-plans.mdx

---
title: Default plans

> description: What are default plans and how they work in Clerk Billing

Reviewer: ```suggestion
description: Learn more about default plans and how they work in Clerk Billing
```

---

<Include src="_partials/billing/billing-experimental" />

When Billing is enabled, Clerk automatically assigns the free plan to every new user or organization as the default plan. This ensures every customer always has an active subscription.

Users are moved to the free plan when they:

- Downgrade from a paid subscription
- Stop paying for a paid subscription
- Cancel a paid subscription

## Can I update it?

You can update the name, slug, and public visibility.

Changing public visibility doesn’t change the free subscription functionality; it just hides it from the pricing table.

## Is it possible to change the default plan?

> Not yet, but we plan to make it configurable. Once implemented, you could set a paid plan (usually with a free trial) as the default plan.

Reviewer: ```suggestion
Not yet, but support for configuring a default plan is planned. This will allow you to set a paid plan (typically with a free trial) as the default.
```
